### PR TITLE
[Reproduce tool] Varied cleanup.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
- Show warnings for flaky stacks/non-reproducible test cases.
- Improve authentication prompt.
- Add an argument to enable verbose logging and disable it by default.
- Add an option to skip Android device setup.
- Improve several prompts and log messages.
- Provide faster feedback by showing output after the initial run.

### CODE REVIEW

File: src/local/butler/reproduce.py [hunk 1]
```diff
@@ -236,7 +235,8 @@ def _update_directory(directory_name, ignore_paths=None):
   environment.set_value('REPRODUCE_TOOL', 'True')
 
   # Force logging to console for this process and child processes.
-  environment.set_value('LOG_TO_CONSOLE', 'True')
+  if verbose:
+    environment.set_value('LOG_TO_CONSOLE', 'True')
```
Comment (mbarbella-chromium) [inline src/local/butler/reproduce.py]: I'm not sure these log messages make sense to the average user. I tried to improve the output when running a bit to give the user faster feedback and set expectations that repeated runs will take a while.

**Comment by mbarbella-chromium** [comment]
PTAL when you have time. This is a collection of small, unrelated changes intended to make the reproduce tool more user friendly.

File: src/local/butler/reproduce.py (hunk 1 above)
Comment (inferno-chromium) [reply src/local/butler/reproduce.py]: s/'True'/True in all places.

Comment (mbarbella-chromium) [reply src/local/butler/reproduce.py]: Done!

**Pull request merged**